### PR TITLE
Add equilibrium test

### PR DESCRIPTION
**Pull request by wirednkod**

### CODE REVIEW

File: tests/paras/paras.toml [hunk 1]
```diff
@@ -32,7 +32,16 @@ chain = "bifrost-kusama-local"
   command = "bifrost"
   args = ["-lparachain=debug"]
 
+[[parachains]]
+id = 2003
+chain = "turing-dev"
```
Comment (pepoviola) [inline tests/paras/paras.toml:37]: This should be a diff chain ( I think they use `dev`).

File: tests/paras/paras.toml [hunk 2]
```diff
@@ -32,7 +32,16 @@ chain = "bifrost-kusama-local"
...
+cumulus_based = true
 
+  [parachains.collator]
+  name = "alicia"
+  image = "{{EQUIL_COL_IMAGE}}"
+  command = "/usr/local/bin/paranode"
+  args = ["--chain", "/etc/chainspec.json", "-d", "/data", "-lparachain=debug"]
```
Comment (pepoviola) [inline tests/paras/paras.toml:44]: why we are using this `chainspec.json` arg?

File: tests/paras/paras.toml [hunk 3]
```diff
@@ -32,7 +32,16 @@ chain = "bifrost-kusama-local"
...
+[[parachains]]
+id = 2003
+chain = "dev"
+cumulus_based = true
 
+  [parachains.collator]
+  name = "alicia"
+  image = "{{EQUIL_COL_IMAGE}}"
+  command = "/usr/local/bin/paranode"
+  args = ["--chain", "/etc/chainspec.json", "-d", "/data", "-lparachain=debug"]
```
Comment (pepoviola) [inline tests/paras/paras.toml:44]:
```suggestion
  args = ["-lparachain=debug"]
```

**Review by l0r1s — APPROVED** [review]

**Review by l0r1s — APPROVED** [review]

**Review by pepoviola — APPROVED** [review]

**Comment by pepoviola** [comment]
@wirednkod this is the error


image

**Pull request closed without merging**